### ISSUE TITLE
Reptile code is not matching the equations in the paper

### ISSUE THREAD

**Issue by wawachen** [resibots/kaushik_2020_famle#1]
hi,

Your work is excellent. But it still has one thing I don't understand.
In the paper, when you optimize the model meta parameter, you provide the **equation 9,10,11** to both optimize the theta and embedding h.
However, in your code, you just update it using simple gradient descent (not reptile), and the embedding h is not optimized.

can you explain that?

**Comment by riturajkaushik** [comment]
Since we used an embedding layer from Pytorch, we don’t have to update the layer parameters separately. Complete network parameters are updated together.

**Comment by riturajkaushik** [comment]
You should be able to find the Reptile update steps here https://github.com/resibots/kaushik_2020_famle/blob/master/fast_adaptation_embedding/models/famle.py#:~:text=model.train(mode%3DFalse,str(np.mean(task_losses)))

**Comment by wawachen** [comment]
ok， I have found them.
Thanks for your explanation!
